### PR TITLE
fix: improve provisioner details layout and show count line

### PR DESCRIPTION
**Pull request by aslilac**
- Make the provisioner tiles more responsive to page width
- Show the number of groups and provisioners at the top of the page
- Better tags handling for PSK groups

### CODE REVIEW

File: site/src/modules/provisioners/ProvisionerGroup.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { type Interpolation, type Theme, useTheme } from "@emotion/react";
 import Business from "@mui/icons-material/Business";
 import Person from "@mui/icons-material/Person";
+import Sell from "@mui/icons-material/Sell";
```
Comment (Parkreiner) [inline site/src/modules/provisioners/ProvisionerGroup.tsx:4]: Could this be renamed to what the icon is meant to indicate?

File: site/src/modules/provisioners/ProvisionerGroup.tsx [hunk 2]
```diff
@@ -343,6 +358,7 @@ const PskProvisionerTags: FC<PskProvisionerTagsProps> = ({ tags }) => {
 				css={{
 					"& .MuiPaper-root": {
 						padding: 20,
+						minWidth: "unset",
```
Comment (Parkreiner) [inline site/src/modules/provisioners/ProvisionerGroup.tsx:361]: Does this do anything? I thought `min-width` wasn't an inheritable property?

**Review by buenos-nachos — APPROVED** [review]
Looks good!
(Also, there's a typo in Jaayden's name – it says "Jyeden", as in rhymes with "chai")

File: site/src/pages/ManagementSettingsPage/OrganizationProvisionersPageView.stories.tsx [hunk 3]
```diff
@@ -53,21 +58,50 @@ export const Provisioners: Story = {
...
+					name: `ケイラ-${i}`,
+				})),
 			},
 		],
 	},
+	play: async ({ step }) => {
+		await step("open all details", async () => {
+			const expandButtons = await screen.findAllByText(
+				"Show provisioner details",
+			);
```
Comment (Parkreiner) [inline site/src/pages/ManagementSettingsPage/OrganizationProvisionersPageView.stories.tsx:84]: Could this be updated to be more accessibility-friendly? I'm just assuming Storybook has a similar API to Testing Library:
```suggestion
const expandedButtons = await screen.findAllByRole("button", {
  name: "Show provisioner details"
});
```

File: site/src/modules/provisioners/ProvisionerGroup.tsx [hunk 4]
```diff
@@ -90,6 +94,10 @@ export const ProvisionerGroup: FC<ProvisionerGroupProps> = ({
 			? "1 provisioner"
 			: `${provisionersWithWarnings} provisioners`;
 
+	const hasMultipleTagVariants =
+		type === "psk" &&
+		provisioners.some((it) => !isEqual(it.tags, { scope: "organization" }));
```
[the remark this answers is not in the document]
Comment (aslilac) [reply site/src/modules/provisioners/ProvisionerGroup.tsx:99]: it's not just comparing the one property, it's also checking for the presence of any other properties

File: site/src/modules/provisioners/ProvisionerGroup.tsx (hunk 2 above)
Comment (aslilac) [reply site/src/modules/provisioners/ProvisionerGroup.tsx:361]: it's not inherited, it's set by MUI

**Comment by aslilac** [comment]
> it says "Jyeden"

何？

In Romaji it should be "Jaiden". I guess it should probably be "Je" but it is definitely not "Jyeden". 🤔

File: site/src/modules/provisioners/ProvisionerGroup.tsx [hunk 5]
```diff
@@ -30,14 +32,16 @@ type ProvisionerGroupType = "builtin" | "psk" | "key";
 
 interface ProvisionerGroupProps {
 	readonly buildInfo?: BuildInfoResponse;
-	readonly keyName?: string;
+	readonly keyName: string;
+	readonly keyTags: Record<string, string>;
 	readonly type: ProvisionerGroupType;
 	readonly provisioners: readonly ProvisionerDaemon[];
 }
```
[the remark this answers is not in the document]
Comment (aslilac) [reply site/src/modules/provisioners/ProvisionerGroup.tsx:39]: yeah, I've just been trying something 🤷‍♀️ idek if we get any benefit out of it when we immediately destructure the props off of the object in the component

File: site/src/modules/provisioners/ProvisionerGroup.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (Parkreiner) [reply site/src/modules/provisioners/ProvisionerGroup.tsx:99]: Oh wait, yeah, that's on me. For some reason, I was thinking of this in terms of TypeScript's structural typing. But this is basically Jest's `toEqual` and not its `expect.objectContaining`

**Comment by Parkreiner** [comment]
@aslilac Yeah, sorry – the "jye" mora doesn't exist in Japanese. I wrote it like that because it's close to the English words "rye" and "dye"

**Comment by aslilac** [comment]
ahh I get what you meant now

**Pull request merged**